airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/integrations/base/ssh/SshWrappedDestination.java

```diff
@@ -98,7 +101,17 @@ public SerializedAirbyteMessageConsumer getSerializedMessageConsumer(final JsonN
                                                                        final ConfiguredAirbyteCatalog catalog,
                                                                        final Consumer<AirbyteMessage> outputRecordCollector)
       throws Exception {
-    final SshTunnel tunnel = getTunnelInstance(config);
+    final JsonNode clone = Jsons.clone(config);
+    Optional<JsonNode> connectionOptionsConfig = Jsons.getOptional(clone, CONNECTION_OPTIONS_KEY);
+    if (connectionOptionsConfig.isEmpty()) {
+      LOGGER.info("No SSH connection options found, using defaults");
+      if (clone instanceof ObjectNode) { // Defensive check, it will always be object node
```

Comment:
nit: you can pattern-match the instanceof to avoid the explicit cast
```java
if (clone instanceof ObjectNode objectClone) {
  ... = objectClone.putObject(...)
```